tox.ini

```diff
@@ -40,5 +40,14 @@ usedevelop = false
 deps = flake8
 commands = flake8 --max-complexity=15
 
+[testenv:docs]
+recreate = false
+skip_install = false
+changedir = docs
+usedevelop = true
+commands =
+    pip install -r requirements.txt
```

Comment:
This really just needs to install all the `extras` so that we get the imports for `autodoc` support